src/clouds.cpp

```diff
@@ -44,7 +48,12 @@ Clouds::Clouds(
 		s16 cloudheight
 ):
 	scene::ISceneNode(parent, mgr, id),
-	m_seed(seed)
+	m_seed(seed),
+	m_camera_pos(0.0f, 0.0f, 0.0f),
```

Comment:
please use proper C++11 constructors instead of old constructors for hardcoded values (see recent PR #5999)